bcrs-ui/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "hello-world",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit --testPathPattern"
```

Comment:
I think this needs to have `--testPathPattern=.`.

Reply:
Hmm.. it actually works as is. It could be just for me locally but i don't think we need to add the `=.` to grab the matching patterns. I actually found by adding that it was pulling more tests into the match outside of the specified test in the coops-ui. Maybe you could try this out and let me know what you think. I know in coops-ui it would pull more tests into the scope than just the matching pattern.

Reply:
You're right, it works like that. I tested this in coops-ui but for me it matched the same tests as before so I dunno.

The reason that parameter is in there is to make it easier for devs (me) to specify a pattern to run just a single test. So I think either we remove it completely (and then I'll have to remember the parameter name when I want to run a single test) or we provide a generic pattern (`.`). Up to you.

Reply:
I'm impartial :) Added it back in!